network-api/app/networkapi/news/apps.py

```diff
@@ -6,5 +6,4 @@ class NewsConfig(AppConfig):
     verbose_name = 'news'
 
     def ready(self):
-        from networkapi.news.signals import setup_signals
-        setup_signals()
+        pass
```

Comment:
we should be able to remove this file entirely, if we remove the contents (but not the file) of network-api/app/networkapi/news/__init__.py